src/foam/dao/Relationship.js

```diff
@@ -23,7 +23,9 @@ foam.CLASS({
   documentation: 'An Axiom for defining Relationships between models.',
 
   requires: [
-    'foam.dao.RelationshipDAO'
+    'foam.dao.RelationshipDAO',
+    'foam.dao.ManyToManyRelationshipDAO',
+    'foam.dao.ReadOnlyDAO',
   ],
 
   properties: [
```

Comment:
Ditto: Even if it's terse, we should take this opportunity to add documentation to these properties.